tests/service/SyntaxTreeTests/MemberTests.fs

```diff
@@ -362,3 +362,26 @@ type X =
         assertRange (4, 13) (4, 19) mInlineGet
         assertRange (5, 12) (5, 18) mInlineSet
     | ast -> Assert.Fail $"Could not get valid AST, got {ast}"
+
+[<Test>]
+let ``ImplicitCtor with as keyword`` () =
```

Comment:
It would be nice to have a clearer name of the test - like what's expected at least. Just a suggestion

Reply:
Please tell me what you want and I can change it 😸.

Reply:
Hmm maybe smth along the lines of "ImplicitCtor range contains as keyword when it's present". Mmm?